### PR TITLE
Fix bug related incorrect error message when over-using empty string

### LINKED ISSUE #672: Incorrect error message when over-using empty string
# Bug report

## What's wrong

Right now:

> hooks/post_gen_project.py:0:1: Z226 Found string constant over-use: 

## How is that should be

> hooks/post_gen_project.py:0:1: Z226 Found string constant over-use: ''

I am using `0.10`

### PR DESCRIPTION
**Pull request by DjaPy**
# I have made things!

## Checklist

- [x] I have double checked that there are no unrelated changes in this pull request (old patches, accidental config files, etc)
- [x] I have created at least one test case for the changes I have made
- [ ] I have updated the documentation for the changes I have made
- [ ] I have added my changes to the `CHANGELOG.md`

## Related issues

Format is:
- Closes #672

### CODE REVIEW

File: tests/test_visitors/test_ast/test_builtins/test_strings/test_overused_string.py [hunk 1]
```diff
@@ -37,10 +37,11 @@ def test_string_overuse(
     default_options,
 ):
     """Ensures that over-used strings raise violations."""
-    tree = parse_ast_tree(string_actions.format(string_value))
+    for string_val in string_values:
```
Comment (sobolevn) [inline tests/test_visitors/test_ast/test_builtins/test_strings/test_overused_string.py]: We use `@pytest.mark.parameterize` instead.
See: https://github.com/wemake-services/wemake-python-styleguide/blob/master/tests/test_visitors/test_ast/test_classes/test_class_attributes/test_slots_syntax.py#L21-L24

Docs: https://docs.pytest.org/en/latest/parametrize.html

**Comment by sobolevn** [comment]
@DjaPy thanks! Please, fix the build and we are ready to go!

**Comment by sobolevn** [comment]
Ok, I fixed it for you. Thanks!

**Comment by DjaPy** [comment]
> Ok, I fixed it for you. Thanks!

Yes, I saw. You beat me to it. Thanks!

**Comment by sobolevn** [comment]
@DjaPy feel free to take #674

**Comment by sobolevn** [comment]
Or #671

**Pull request merged**